src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/ja/VSResources.xlf.lcl

```diff
@@ -73,15 +73,6 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
-      <Item ItemId=";EditProjectFileCommand" ItemType="0" PsrId="308" Leaf="true">
-        <Str Cat="Text">
-          <Val><![CDATA[Edit {0}]]></Val>
-          <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[{0} を編集する]]></Val>
-          </Tgt>
-        </Str>
-        <Disp Icon="Str" />
-      </Item>
```

Comment:
The removals are expected following 1b782f12b0fc016c63fe1a559434259ac75a0285. In fact they should be made to other languages too.